sdk/communication/Azure.Communication.CallAutomation/src/Models/CallInvite.cs

```diff
@@ -15,28 +15,34 @@ public class CallInvite
         /// </summary>
         /// <param name="targetPhoneNumberIdentity"></param>
         /// <param name="callerIdNumber"></param>
-        public CallInvite(PhoneNumberIdentifier targetPhoneNumberIdentity, PhoneNumberIdentifier callerIdNumber)
+        /// /// <param name="sipHeaders"></param>
+        public CallInvite(PhoneNumberIdentifier targetPhoneNumberIdentity, PhoneNumberIdentifier callerIdNumber, IDictionary<string, string> sipHeaders = null)
```

Comment:
can the default just be `new Dictionary<string, string>()` if sipHeaders is not provided?